Buildings/Media/Examples/BaseClasses/FluidProperties.mo

```diff
@@ -19,8 +19,8 @@ equation
     state_pTX = Medium.setState_pTX(p=p, T=T, X=X);
     state_phX = Medium.setState_phX(p=p, h=h, X=X);
     state_psX = Medium.setState_psX(p=p, s=s, X=X);
-    checkState(state_pTX, state_phX, "state_phX");
-    checkState(state_pTX, state_psX, "state_psX");
+    checkState(state_pTX, state_phX, errAbs, "state_phX");
+    checkState(state_pTX, state_psX, errAbs, "state_psX");
```

Comment:
Should modelica-fmt also be run on this model?

Reply:
No, I don't think so. (It also existed in the MBL previously.)